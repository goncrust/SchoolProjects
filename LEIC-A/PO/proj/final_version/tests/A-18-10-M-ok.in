4
3
969015
8
969003
werty
0
3
969003
1
0
0
3
1
cli001
1
cli003
0
0
