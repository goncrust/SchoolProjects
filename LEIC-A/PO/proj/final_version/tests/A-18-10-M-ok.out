MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: Número do terminal: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Mensagem de texto a enviar: O terminal de destino ('969003') está desligado.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: Número do terminal: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: GESTÃO DE CLIENTES
1 - Visualizar cliente
2 - Visualizar todos os clientes
3 - Registar cliente
4 - Activar notificações
5 - Desactivar notificações
6 - Mostrar saldo de cliente
0 - Sair
Escolha uma opção: Identificador do cliente: CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|0
GESTÃO DE CLIENTES
1 - Visualizar cliente
2 - Visualizar todos os clientes
3 - Registar cliente
4 - Activar notificações
5 - Desactivar notificações
6 - Mostrar saldo de cliente
0 - Sair
Escolha uma opção: Identificador do cliente: CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
O2I|969003
GESTÃO DE CLIENTES
1 - Visualizar cliente
2 - Visualizar todos os clientes
3 - Registar cliente
4 - Activar notificações
5 - Desactivar notificações
6 - Mostrar saldo de cliente
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: 